plugins/modules/mount.py

```diff
@@ -105,24 +106,47 @@
 '''
 
 EXAMPLES = r'''
-- name: List mounted filesystems
+- name: List mounted filesytems
   ibm.power_aix.mount:
+    state: list
```

Comment:
Should this be state: show?

Reply:
yes you are correct. i will make changes regarding that